styles/sass/partials/_popup.scss

```diff
@@ -0,0 +1,53 @@
+.popup__container--closed {
+  background: url("../assets/jakob-braun-bricks.jpg") no-repeat center / cover fixed;
+  filter: grayscale(70%) brightness(90%);
```

Comment:
filter is currently applying this to the div this class is applied to AND all of its children, which means all the text is also affected by it. Unfortunately, this makes the popup difficult to read, which is an accessibility concern.

If you want it to only apply to the background, you can use a pseudo element like I did for the body background.